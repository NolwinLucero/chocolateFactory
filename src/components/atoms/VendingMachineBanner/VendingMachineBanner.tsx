import React from 'react';

export const VendingMachineBanner: React.FC = () => {
  return (
    <>
      <h1> Brisbane's Chocolate Factory</h1>
      <div>Allowed amounts: 10c, 20c, 50c, $1, $2</div>
    </>
  );
};
