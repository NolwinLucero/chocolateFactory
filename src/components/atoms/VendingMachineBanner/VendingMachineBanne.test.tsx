import React from 'react';
import ReactDOM from 'react-dom';
import { VendingMachineBanner } from '../VendingMachineBanner/VendingMachineBanner';
import renderer from 'react-test-renderer';

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<VendingMachineBanner></VendingMachineBanner>, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('matches snapshot', () => {
  const tree = renderer
    .create(<VendingMachineBanner></VendingMachineBanner>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
