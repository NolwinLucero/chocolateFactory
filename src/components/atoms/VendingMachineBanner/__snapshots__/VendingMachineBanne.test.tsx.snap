// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
Array [
  <h1>
     Brisbane's Chocolate Factory
  </h1>,
  <div>
    Allowed amounts: 10c, 20c, 50c, $1, $2
  </div>,
]
`;
